llvm/include/llvm/MC/MCDwarf.h

```diff
@@ -634,6 +649,22 @@ class MCCFIInstruction {
     return MCCFIInstruction(OpRegister, L, Register1, Register2, Loc);
   }
 
+  /// This is a "pseudo CFI" instruction which generates the escape expression
+  /// deref(FrameReg + Offset) for the register Reg.
+  static void createRegOffsetExpression(unsigned Reg, unsigned FrameReg,
```

Comment:
This function writes the result into CFAExpr. I think you should clarify this in the docstring. And perhaps you should document what it writes into CFAExpr.